microsoft/testsuites/nested/nested.py

```diff
@@ -0,0 +1,105 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import time
+from typing import Any, Dict
+
+from assertpy import assert_that
+
+from lisa import RemoteNode, TestCaseMetadata, TestSuite, TestSuiteMetadata
+from lisa.schema import Node
+from lisa.tools import Cat, Echo, Qemu, Sshpass, Wget
+
+
+@TestSuiteMetadata(
+    area="nested",
+    category="functional",
+    description="""
+    This test suite is used to run nested vm related tests.
+    """,
+)
+class Nested(TestSuite):
+    NESTED_VM_TEST_FILE_NAME = "message.txt"
+    NESTED_VM_TEST_FILE_CONTENT = "Message from L1 vm!!"
+    NESTED_VM_TEST_PUBLIC_FILE_URL = "http://www.github.com"
+
+    def get_l2_vm(
+        self,
+        host: RemoteNode,
+        guest_username: str,
+        guest_password: str,
+        guest_port: int,
+        guest_image_url: str,
+    ) -> RemoteNode:
+        # start nested vm
+        host.tools[Qemu].create_nested_vm(guest_port, guest_image_url)
+
+        # wait for the VM to boot
+        # ToDo: Find a more robust solution
+        time.sleep(60)
```

Comment:
The following steps can wait it start. Don't put any hardcode sleep in the code.